app/src/main/java/com/breadwallet/tools/security/BRSender.java

```diff
@@ -288,18 +288,10 @@ public void run() {
                 BRExecutor.getInstance().forMainThreadTasks().execute(new Runnable() {
                     @Override
                     public void run() {
-                        PostAuth.getInstance().onPublishTxAuth(ctx, false);
```

Comment:
Oh wow. Thanks @andhikayuana .  I'm sure I was guilty of poor conflict resolution